content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,53 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
+
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to setup and deploy to additional platforms.
+
+By default, the Asset Processor will create a worker job for each logical core on the host machine. If you wish to manage this yourself, you can adjust the setting `Amazon/AssetProcessor/Settings/Jobs/maxJobs`. A value of zero will have the Asset Processor create a job for each logical core on your machine. A value of one can be useful when debugging, if you wish to attach a debugger to a running builder processing a specific asset. Other values lower than your logical core count can be useful to manage the impact asset processing has on your machine while you are trying to perform other tasks, with the tradeoff of a longer time before all assets are processed.
+
+The asset cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`. Documentation for this feature is still in progress, feel free to reach out on Discord if you have questions about the Asset Cache Server in the meantime.
+
+Meta data file types for assets are managed under the setting path `Amazon/AssetProcessor/Settings/MetaDataTypes`, with a format of meta data extension : associated file extension, for example FBX files use a meta data file type with the extension fbx.assetinfo, which is defined under this setting path as "fbx.assetinfo": "fbx". If you are adding a new asset type and want to use a metadata file to track some settings for the new asset type, make sure to add it to this setting path, so the Asset Processor will re-run the processing job for the source asset when the meta data file changes.
+
+Scan folders can be added as registry settings. In most cases you'll use the asset folders of Gems or your project, which will be automatically added as scan folders. However, if you need to add a new scan folder for tracking source assets, you can add it under the registry setting path `Amazon/AssetProcessor/Settings/` with a pattern of:
+```json
+"ScanFolder _scan folder name here_": {
+  "watch": "_path to watch folder_",
+  "display": "_display name for scan folder_",
+  "recursive": 0 or 1, // 1 will tell the Asset Processor to look in sub folders, recursively for additional source assets. 0 it will not.
+  "order": any number // The order this scan folder is handled compared to other scan folders. 

```

Comment:
Might want to add: lower numbers are higher priority, higher numbers are lower priority.

This mostly affects the override system, files in higher priority scanfolders override those in lower priority scanfolders

Reply:
Done